src/quacc/calculators/espresso/espresso.py

```diff
@@ -71,6 +71,8 @@ def __init__(
 
         self.binary = binary
 
+        self.ase_known = self.binary in ALL_KEYS
```

Comment:
Maybe change to `self.known_binary` or `self.ase_binary` or something? I find `self.ase_known` a bit cryptic. I'd also keep this as a hidden attribute with a leading `_` since this is not really something we want or expect users to query based on.

Reply:
Will do!